templates/kafka/0/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+zk:
```

Comment:
@rawmind0, for now adding it to the community catalog would be good. The next release has a feature that will allow us to keep quorum in a ZK cluster during an upgrade, and we will be revisiting the template again. See rancher/rancher#2494. I'll be looking at your template features as we roll out the next iteration of the Rancher version.

In the mean time the community could benefit from the ZK image.